woocommerce/woocommerce
# Product Collection: Handle empty block content in process_pagination_links method

### Changes proposed in this Pull Request:





Closes #44574



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Enable debugging
2. Add a Product Collection block to a new post & choose any collection. 
3. Set the `MAX PAGE TO SHOW` to 1.
4. Publish & go to frontend. Verify that there shouldn’t be following notice: 

```jsx
Notice: Function WP_HTML_Tag_Processor::seek was called incorrectly. Unknown bookmark name. Please see <a>Debugging in WordPress</a> for more information. (This message was added in version 6.2.0.)
```


### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

Fix following notice on frontend:
`Notice: Function WP_HTML_Tag_Processor::seek was called incorrectly. Unknown bookmark name. Please see <a>Debugging in WordPress</a> for more information. (This message was added in version 6.2.0.)`

</details>
